Competition2018/src/test/java/competition/subsystems/gripperintake/commands/GripperStopCommandTest.java

```diff
@@ -0,0 +1,38 @@
+package competition.subsystems.gripperintake.commands;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+import competition.BaseCompetitionTest;
+import competition.subsystems.gripperintake.GripperIntakeSubsystem;
+
+public class GripperStopCommandTest extends BaseCompetitionTest {
+
+    GripperStopCommand command;
+    GripperIntakeSubsystem gripper;
+    
+    @Override
+    public void setUp() {
+        super.setUp();
+        
+        command = injector.getInstance(GripperStopCommand.class);
+        gripper = injector.getInstance(GripperIntakeSubsystem.class);
+    }
+    
+    @Test
+    public void simpleTest() {
+        command.initialize();
+        command.execute();
+    }
+    
+    @Test
+    public void verifyPower() {
```

Comment:
You need to have the power already be some value (say 1), verify that value, then run the stop command, and verify it stopped.